### PR TITLE
small improvements to the encrypted secret doc

### PR DESCRIPTION
**Pull request by arbll**
### What does this PR do?

A few improvements that I think would be nice to have.

### CODE REVIEW

File: docs/agent/secrets.md [hunk 1]
```diff
@@ -247,3 +259,30 @@ instances:
...
+Source: File Configuration Provider
+Instance 1:
+host: <decrypted_host2>
+port: <decrypted_port2>
+passowrd: <decrypted_password2>
+~
+===
+```
+
+Note that the agent needs to be restarted to pick up new configuration changes.
```
Comment (olivielpeau) [inline docs/agent/secrets.md]: `to pick up new configuration changes` -> `to pick up changes on config files` (just want to mention that it needs to be restarted for files but not for other config sources)

**Review by olivielpeau — DISMISSED** [review]
🚀 

just a nit

let's wait for a review from @hush-hush too :)

File: docs/agent/secrets.md [hunk 2]
```diff
@@ -247,3 +259,30 @@ instances:
...
+passowrd: <decrypted_password>
+~
+===
+
+=== another check ===
+Source: File Configuration Provider
+Instance 1:
+host: <decrypted_host2>
+port: <decrypted_port2>
+passowrd: <decrypted_password2>
```
Comment (ngetahun-dd) [inline docs/agent/secrets.md]: ~passowrd~ password

File: docs/agent/secrets.md [hunk 3]
```diff
@@ -247,3 +259,30 @@ instances:
...
+
+```shell
+sudo -u dd-agent -- datadog-agent configcheck
+
+=== a check ===
+Source: File Configuration Provider
+Instance 1:
+host: <decrypted_host>
+port: <decrypted_port>
+passowrd: <decrypted_password>
```
Comment (ngetahun-dd) [inline docs/agent/secrets.md]: ~passowrd~ password

**Review by hush-hush — APPROVED** [review]

**Pull request merged**